src/components/CDPDisplay/Presentation.js

```diff
@@ -48,9 +48,9 @@ export default function({ cdp, showSidebar, account, network, cdpOwner }) {
     .getContractAddress('CDP_MANAGER');
 
   // Testing:
-  const { dsProxy } = useObservable('dsProxy', account?.address);
-  const { getVaults } = useObservable('getVaults', manager, dsProxy);
-  const { ilkPrice } = useObservable('ilkPrice', 'ETH-A');
+  const { dsProxy } = watch.dsProxy(account?.address);
+  const { getVaults } = watch.getVaults(manager, dsProxy);
+  const { ilkPrice } = watch.ilkPrice('ETH-A');
```

Comment:
Dude, this is slick! Importing the schemas from the plugin and adding into the watcher in MakerProvider works really nice too.